--- openvpn-2.4.8/version.m4
+++ openvpn-2.4.8/version.m4
@@ -10,6 +10,6 @@
 define([PRODUCT_BUGREPORT], [openvpn-users@lists.sourceforge.net])
 define([PRODUCT_VERSION_RESOURCE], [2,4,8,0])
 dnl define the TAP version
-define([PRODUCT_TAP_WIN_COMPONENT_ID], [tap0901])
+define([PRODUCT_TAP_WIN_COMPONENT_ID], [tapprotonvpn])
 define([PRODUCT_TAP_WIN_MIN_MAJOR], [9])
 define([PRODUCT_TAP_WIN_MIN_MINOR], [9])
